uxarray/grid/grid.py

```diff
@@ -119,6 +119,9 @@ def __init__(self,
         self._gdf = None
         self._poly_collection = None
         self._line_collection = None
+        self._centroid_points_df_proj = [None, None]
```

Comment:
Is this documented?
what is `..df_proj`?

Reply:
it's internally used in our point raster for caching, it's not referenced anywhere else

It's a list containing our Point Data Frame and Projection respectively.